space-cli/cmd/modules/addons/database.go

```diff
@@ -32,6 +32,13 @@ func addDatabase(dbtype, username, password, alias, version string) error {
 	ctx := context.Background()
 	autoApply := viper.GetBool("auto-apply")
 	project := viper.GetString("project")
+	clusterName := viper.GetString("cluster-id")
+
+	// change selected account according to cluster name provided
+	if err := utils.ChangeSelectedAccount(clusterName); err != nil {
+		return utils.LogError("Could not set selected account ", err)
+	}
```

Comment:
@sharadregoti instead of changing the selected account here, why don't we add the database to the selected account be default? We wont need the `cluster-name` flag in that case.

Reply:
the problem is when we add a second cluster the selected account changes to that cluster-id and suppose for this case we wish to add a database to the previously set cluster we have to change the selected account to that cluster-id which has to be done manually. I and @sharadregoti  decided that we will automatically do that if the cluster-name is provided.

Reply:
What if the clustername is not provided? Will you use the currently selected cluster or the default cluster? Ideally it should be the currently selected cluster